client/extensions/woocommerce/index.js

```diff
@@ -14,31 +14,88 @@ import ProductCreate from './app/products/product-create';
 import Dashboard from './app/dashboard';
 import StatsController from './app/stats/controller';
 
-const Controller = {
-	dashboard: function( context ) {
+function addStorePage( storePage ) {
+	page( storePage.route, siteSelection, navigation, function( context ) {
```

Comment:
ya know, since store stores isn't really part of the store and doesn't use store nav, i'm going to move its page magic outside of this logic - we can always bring it back in later if we want